src/plugins/columnSorting/columnSorting.js

```diff
@@ -70,35 +104,22 @@ class ColumnSorting extends BasePlugin {
 
     this.setPluginOptions();
 
-    const _this = this;
-    this.hot.sortIndex = [];
-
-    this.hot.sort = function() {
-      let args = Array.prototype.slice.call(arguments);
-
-      return _this.sortByColumn(...args);
-    };
-
-    if (typeof this.hot.getSettings().observeChanges === 'undefined') {
+    if (isUndefined(this.hot.getSettings().observeChanges)) {
       this.enableObserveChangesPlugin();
     }
 
-    this.addHook('afterTrimRow', (row) => this.sort());
-    this.addHook('afterUntrimRow', (row) => this.sort());
-    this.addHook('modifyRow', (row) => this.translateRow(row));
-    this.addHook('unmodifyRow', (row) => this.untranslateRow(row));
+    this.addHook('afterTrimRow', () => this.sortByPresetColumnAndOrder());
+    this.addHook('afterUntrimRow', () => this.sortByPresetColumnAndOrder());
+    this.addHook('modifyRow', (row, source) => this.onModifyRow(row, source));
```

Comment:
You can use just `...args` instead of a list of arguments.

Reply:
For me it's ok.